src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -6,12 +6,14 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
+  <ItemGroup Condition="'$(TargetGroup)' != 'uapaot'">
+    <Compile Include="$(TestSourceFolder)DataContractJsonSerializer.CoreCLR.cs" />
```

Comment:
Do we still want to run them in UAP? that mens corefx going against CoreCLR as well... I ask because if we don't want to run them there either, then this should be a uap configuration instead of uapaot